src/services/passwordRecovery/services/ChangePasswordService.js

```diff
@@ -3,14 +3,14 @@ const local = require('@feathersjs/authentication-local');
 const { BadRequest, GeneralError, SilentError } = require('../../../errors');
 const logger = require('../../../logger/index');
 const globalHooks = require('../../../hooks');
-const { ObjectId } = require('../../../helper/compare');
 
 const MAX_LIVE_TIME = 6 * 60 * 60 * 1000; // 6 hours
 
 class ChangePasswordService {
-	constructor(passwordRecoveryModel, accountModel) {
+	constructor(passwordRecoveryModel, app) {
```

Comment:
The ChangePasswordService only needs the AccountService, not the whole app.

Reply:
Checked ok. We need the reference and make use of it only after the app is fully initialized to make the service call.